projects/kit/src/lib/masks/date-range/date-range-mask.ts

```diff
@@ -19,13 +19,15 @@ export function maskitoDateRangeOptionsGenerator({
     max,
     minLength,
     maxLength,
+    rangeSeparator = DEFAULT_DATE_RANGE_SEPARATOR,
```

Comment:
```suggestion
    rangeSeparator = `${CHAR_NO_BREAK_SPACE}${CHAR_EN_DASH}${CHAR_NO_BREAK_SPACE}`,
```

\+ delete `DEFAULT_DATE_RANGE_SEPARATOR`

Reply:
fixed